index.bs

```diff
@@ -195,6 +195,8 @@ namespace console { // but see namespace object requirements below
   void table(any tabularData, optional sequence&lt;DOMString> properties);
   void trace(any... data);
   void warn(any... data);
+  void dir(any data);
```

Comment:
Went with `item` here, since in `dirxml` when I iterate over `data` I call individual elements `item`s